app/degreePlanner/degreePlanner.css

```diff
@@ -40,3 +41,7 @@
 .formElement .degreeSelect:invalid {
     color: #999;
 }
+
+#degree.degreeSelect,#firstMajor.degreeSelect,#secondMajor.degreeSelect,#minor.degreeSelect{
```

Comment:
You can use existing css to change the width instead of making a new rule set.

Reply:
Thank you for the feedback!

I've updated the implementation to use the existing CSS rules to change the width of the inputs instead of creating new rule sets. Please review the changes and let me know if there are any further adjustments needed.

Reply:
@jennifermtan Please resolve conversation if they have been fixed.

Reply:
@GoodGameRuler Could you authorise Vercel for this PR so I can check before I approve? Thanks!

Reply:
Will authorise now, but you can also just checkout into this branch. @jennifermtan @MehekFatima